app.js

```diff
@@ -95,7 +100,7 @@ async function formSubmit(event) {
       const type = response.headers.get("content-type");
       const contentType = parseContentType(type);
       const text = await response.text();
-
+      
```

Comment:
Please remove extra whitespace

Reply:
Isn't this line break set by eslint ```newline-after-var``` ?

Reply:
The newline is OK, but it seems to contain a bunch of spaces (`...await response.text();` linebreak, space, space, space ... linebreak). Instead, we want (`...await response.text();`, linebreak, linebreak).

We call these "extra whitespaces" or trailing whitespace